flink-libraries/flink-python/src/test/java/org/apache/flink/python/api/PythonPlanBinderUnitTest.java

```diff
@@ -87,13 +85,10 @@ private static String findSubproessTestCaseFile() throws Exception {
 			if (verbose) {
 				System.out.println("cwd: " + cwd);
 			}
-
-			String pythonPath = cwd + "/src/main/python/org/apache/flink/python/api/" + ":" +
```

Comment:
This caused issues on windows since ```:``` is always part of absolute paths. When this path is later split in python it creates unusable paths.